gravitee-apim-console-webui/src/portal/portal-settings-routing.module.ts

```diff
@@ -114,6 +116,21 @@ const portalRoutes: Routes = [
               },
             },
           },
+          {
+            path: 'api',
+            component: PortalApiComponent,
+            children: [
+              {
+                path: '',
+                component: PortalApiListComponent,
+                data: {
+                  permissions: {
+                    anyOf: ['environment-api-r', 'environment-api-u'],
```

Comment:
Should be settings permissions, not apis

Reply:
Done